playbooks/06-deploy-architecture.yml

```diff
@@ -197,6 +197,15 @@
         - update_containers
         - edpm_bootstrap
 
+    - name: Update containers in deployed OSP operators using set_openstack_containers role
+      vars:
+        cifmw_set_openstack_containers_extra_vars: "{{ cifmw_edpm_prepare_extra_vars }}"
+      ansible.builtin.include_role:
+        name: set_openstack_containers
+      tags:
+        - set_openstack_containers
```

Comment:
A few comments:
- You assume we call deploy-edpm only using the reproducer, that with this change, will inject the tag in the skip list
- You assume that we want the role to be executed by default, that's the meaning of adding a tag here without a when condition.
- This skip strategy is not aligned with the rest of "skip flags" we have all over the framework. Please, use a flag like `cifmw_update_containers` with a sane `false` default.

Reply:
@ciecierski sorry mate, I may miss-expressed what I wanted. `cifmw_update_containers` was just an example of the kind of flag we usually do, for this new task create a new one called: `cifmw_set_openstack_containers` that defaults to false. Yes, false, as the regular behavior we had (and we still want) is to preserve what's deployed. If someone wants to patch the images all he/she needs is to make the flag true.

Reply:
PS: We doc those vars [here](https://github.com/openstack-k8s-operators/ci-framework/blob/main/docs/source/usage/01_usage.md), so add the new one in that file too.

Reply:
@pablintino I added cifmw_update_containers to not create more vars, but then I realized that there are deployments jobs which have cifmw_update_containers set to false. Those deployment jobs are using containers from compose. I'm going to add cifmw_set_openstack_containers for  set_openstack_containers role include. 
I'm not sure if I should remove cifmw_update_containers from update_containers include. update_containers role default value for  cifmw_update_containers is false, but it used only for applying generated template: https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/update_containers/tasks/main.yml#L30
Do you think we want to generate template always? Should I remove when: cifmw_update_containers or it can stay?

Reply:
@ciecierski I thought about telling you to do so, but I'd prefer to keep your change as scoped to your issue as posible, so no worries, we will do it during the cleanup of the framework.
Care to add the new var to this doc? https://github.com/openstack-k8s-operators/ci-framework/blob/main/docs/source/usage/01_usage.md